sources/vega/api/v1/core.proto

```diff
@@ -281,6 +281,18 @@ message LastBlockHeightRequest {}
 // A response with the height of the last block processed by
 // tendermint
 message LastBlockHeightResponse {
+  // last block height
   uint64 height = 1;
+  // last block hash
   string hash = 2;
+  // supported proof of work hash function
+  string spam_pow_hash_function = 3;
+  // the difficulty of the proof of work (target number of zeros)
+  uint64 spam_pow_difficulty = 4;
```

Comment:
This can be the smallest `uint` available, which is `uint32` (as per slack)

Reply:
will change